internal/terraform/aws/dyanmodb_table.go

```diff
@@ -0,0 +1,114 @@
+package aws
+
+import (
+	"fmt"
+	"infracost/pkg/resource"
+
+	"github.com/shopspring/decimal"
+	log "github.com/sirupsen/logrus"
+)
+
+func NewDynamoDBGlobalTable(address string, region string, rawValues map[string]interface{}) resource.Resource {
+	r := resource.NewBaseResource(address, rawValues, true)
+
+	hoursProductFilter := &resource.ProductFilter{
+		VendorName:    strPtr("aws"),
+		Region:        strPtr(region),
+		Service:       strPtr("AmazonDynamoDB"),
+		ProductFamily: strPtr("DDB-Operation-ReplicatedWrite"),
+		AttributeFilters: &[]resource.AttributeFilter{
+			{Key: "group", Value: strPtr("DDB-ReplicatedWriteUnits")},
+		},
+	}
+	hoursPriceFilter := &resource.PriceFilter{
+		PurchaseOption:   strPtr("on_demand"),
+		DescriptionRegex: strPtr("/beyond the free tier/"),
+	}
+	rwcu := resource.NewBasePriceComponent("Replicated write capacity unit (rWCU)", r, "hour", "hour", hoursProductFilter, hoursPriceFilter)
+	rwcu.SetQuantityMultiplierFunc(dynamoDBWCUQuantity)
+	r.AddPriceComponent(rwcu)
+
+	return r
+}
+
+func dynamoDBWCUQuantity(resource resource.Resource) decimal.Decimal {
+	quantity := decimal.Zero
+
+	capacity := resource.RawValues()["write_capacity"]
+	if capacity != nil {
+		quantity = decimal.NewFromFloat((capacity.(float64)))
+	}
+	return quantity
+}
+
+func dynamoDBRCUQuantity(resource resource.Resource) decimal.Decimal {
+	quantity := decimal.Zero
+
+	capacity := resource.RawValues()["read_capacity"]
+	if capacity != nil {
+		quantity = decimal.NewFromFloat((capacity.(float64)))
+	}
+	return quantity
+}
+
+func NewDynamoDBTable(address string, region string, rawValues map[string]interface{}) resource.Resource {
+	r := resource.NewBaseResource(address, rawValues, true)
+
+	// Check billing mode
+	billingMode := rawValues["billing_mode"]
+	if billingMode != "PROVISIONED" {
```

Comment:
@aliscott I couldn't figure out a suitable way to test this case in the integration test. Any help will be appreciated.

Reply:
Yeah I was thinking about this as well for adding functionality to ensure all integrations tests log no warnings or errors. It seems logrus has some hooks we might be able to use: https://github.com/sirupsen/logrus#testing. I wouldn't block this PR on this though. I'll create a separate issue and add this to the list of tests to include in it.